src/packaging/kpatch/Makefile.upstream

```diff
@@ -1,50 +1,39 @@
 # SPDX-License-Identifier: GPL-2.0-only
 
+ccflags-$(CONFIG_DM_VDO) := -I${src} -I${src}/indexer
```

Comment:
I think this should be ccflags-y, based on reading Documentation/kbuild/makefiles.rst .

The ccflags-$(CONFIG_FOO) style works for optional flags wanted depending on some additional boolean y/n flag like CONFIG_ACPI_DEBUG. But CONFIG_DM_VDO is tristate and can be y/m/n.

Reply:
*sigh* Fixed. I wish the documentation were a little clearer about that.